messaging/WFLY-11293_add_attribute_to_indicate_backup_is_synchronized.adoc

```diff
@@ -0,0 +1,57 @@
+= Add attribute to model which will indicate that backup is synchronized with live server
+:author:            Emmanuel Hugonnet
+:email:             ehugonne@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:          messaging,jms,ha,cluster
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11293[WFLY-11293]
+
+=== Related Issues
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[ ] Engineering
```

Comment:
You've already prepared the tests so I guess this is tested by engineering, right?